noria-benchmarks/lobsters/src/main.rs

```diff
@@ -15,12 +15,16 @@ use trawler::{LobstersRequest, UserId};
 const ORIGINAL_SCHEMA: &'static str = include_str!("../db-schema/original.sql");
 const NORIA_SCHEMA: &'static str = include_str!("../db-schema/noria.sql");
 const NATURAL_SCHEMA: &'static str = include_str!("../db-schema/natural.sql");
+const CAN_COMBINE_SCHEMA: &'static str = include_str!("../db-schema/cancombine.sql");
+const FILTER_AGGREGATE_SCHEMA: &'static str = include_str!("../db-schema/filteraggregate.sql");
```

Comment:
How do these differ from the other ones?

Reply:
Originally I thought this was how I would go about testing things and making graphs. Then Malte pointed me at adding a test to integration.rs, and I forgot to remove the added schema here. I'll do that now.